src/components/FindPark.vue

```diff
@@ -1,7 +1,10 @@
 <template>
   <section>
     <h2>Let's Go Play!</h2>
-    <router-link to='/results'>
+    <router-link
```

Comment:
I'm confused why this wasn't working this morning, but I'm happy to see it working now! Maybe we were just having a syntax error before or something. This is an ideal user-experience.